resources/oe/src/oeCore.i

```diff
@@ -327,7 +351,8 @@ FUNCTION GET_COLUMNS RETURNS Progress.Json.ObjectModel.JsonArray (lDumpFile AS L
     DEFINE VARIABLE qhField AS HANDLE NO-UNDO.
     DEFINE VARIABLE bhField AS HANDLE  NO-UNDO.
 
-    DEFINE VARIABLE iFieldCount AS INTEGER NO-UNDO.
+    DEFINE VARIABLE iFieldExtentCount AS INTEGER NO-UNDO.
+    DEFINE VARIABLE i AS INTEGER NO-UNDO.
```

Comment:
Try to avoid `i` as a variable name. Replace with `iCount`.

Reply:
ok. I`ll update it.